cvxpy/tests/test_python_backends.py

```diff
@@ -854,7 +860,9 @@ def test_parametrized_mul(self, backend):
         assert np.all(slice_idx_one == expected_idx_one)
 
         # indices are: variable 1, parameter 2, 2 index of the list
-        slice_idx_two = out_view.tensor[1][2][2].toarray()
+        slice_idx_two = out_view.tensor[1][2][2]
+        if not isinstance(slice_idx_two, np.ndarray):
```

Comment:
I added this instance check because the numpy backend triggered errors when we called .toarray() on a ndarray. I verified that this has no effect on the original scipy backend test.